docs/UserGuide.md

```diff
@@ -544,16 +579,22 @@ Note that the last example is **not equivalent** to `n/do && t/friend || t/colle
 # Planned Enhancements / Known Issues
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
+2. **When executing `addalt` command**, if you input other prefixes that is not accepted by the command format, the error message shown does not prompt you to remove those prefixes and adhere strictly to the command format. We will be working on this in the future to improve the specificity of error messages.
```

Comment:
```suggestion
2. **When executing `addalt` command**, if you input other prefixes that are not accepted by the command format, the error message shown does not prompt you to remove those prefixes and adhere strictly to the command format. We will be working on this in the future to improve the specificity of error messages.
```

Reply:
nice catch of grammatical errors, fixed in b33ffa82